### PR TITLE
Fix python path issue in cmake-converter

### PR DESCRIPTION
**Pull request by nicklauslittle**

### CODE REVIEW

File: src/pkgs/cmake-converter.ps1 [hunk 1]
```diff
@@ -28,9 +28,14 @@ function global:Install-PwrPackage {
 	Invoke-WebRequest -UseBasicParsing 'https://bootstrap.pypa.io/get-pip.py' -OutFile '\python\get-pip.py'
 	& '\python\python.exe' '\python\get-pip.py' --no-warn-script-location
 	& '\python\python.exe' -m pip install '--target' '\pkg' $env:repo
+	# Fix python path in exe
+	$ExeDirectory = (Get-ChildItem -Path '\pkg' -Recurse -Include 'cmake-converter.exe' | Select-Object -First 1).DirectoryName
+	$ExeData = Get-Content -Raw -AsByteStream "$ExeDirectory\cmake-converter.exe"
+	$FixedData = (-split (($ExeData.ForEach('ToString', 'X') -join ' ') -replace '\b23 21( [2-7].)+ 5C 70 79 74 68 6F 6E 2E 65 78 65', '23 21 70 79 74 68 6F 6E 2E 65 78 65') -replace '^', '0x') -as [byte[]]
```
Comment (nicklauslittle) [inline src/pkgs/cmake-converter.ps1:34]: [Fully expecting barf emoji below.]

**Comment by nicklauslittle** [comment]
Another thought: may be able to use `pwr shell python` to install this instead of manually downloading now.

File: src/pkgs/cmake-converter.ps1 (hunk 1 above)
Comment (troyaf) [reply src/pkgs/cmake-converter.ps1:34]: 🤮

**Comment by troyaf** [comment]
> Another thought: may be able to use `pwr shell python` to install this instead of manually downloading and installing python now that the install script pre-installs `pwr`.

Sounds good

File: src/pkgs/cmake-converter.ps1 (hunk 1 above)
Comment (nicklauslittle) [reply src/pkgs/cmake-converter.ps1:30]: I really didn't want to do this. I searched for a while to see if I could find a better way. Let me know if you know of a better way to do this in PS or if you have another idea on how to handle the absolute path.

**Pull request merged**